regex-assembly/include/unix-shell-evasion-prefix.ra

```diff
@@ -40,6 +41,8 @@
 >\(
 ##! a() ( ifconfig; ); a
 \(\s*\)
+##! `cat<<<ifconfig` or `cat<<< ifconfig`
+<<<
```

Comment:
You might be missing optional white space here, not sure. Please add a couple of tests as this affects about 6 rules.